v2/src/plugins/codeTypeChecking/index.js

```diff
@@ -1,10 +1,10 @@
-let fs = require('fs');
```

Comment:
such a huge diff is hard to review. Please make sure only the parts you have changed show up.

Reply:
done